### PR TITLE
ref(makefile): Update makefile to work with new chart structure

### PR DESCRIPTION
**Pull request by jchauncey**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,26 +7,19 @@ include versioning.mk
...
-
-kube-install: update-manifests
-	kubectl create -f manifests/deis-logger-fluentd-daemon.tmp.yaml
+test: docker-build
+	docker run ${IMAGE} /bin/bash -c "cd /opt/fluentd/deis-output && rake test"
 
-kube-update: update-manifests
-	kubectl delete -f manifests/deis-logger-fluentd-daemon.tmp.yaml
-	kubectl create -f manifests/deis-logger-fluentd-daemon.tmp.yaml
+install:
```
Comment (vdice) [inline Makefile:18]: looks like `install` target does same as `upgrade` target -- if so maybe just have `install: upgrade` ?

**Review by vdice — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (jchauncey) [reply Makefile:18]: no install performs a `--install` with the helm upgrade command. but upgrade does not. one if for installing a fresh chart the other is to update the daemonset

Comment (vdice) [reply Makefile:18]: ah makes sense.  just curious -- shouldn't the `helm upgrade --install ...` command (`upgrade` target) then work for both cases?  (Would install if release name not previously existing and would upgrade if it did exist)

Comment (jchauncey) [reply Makefile:18]: No `--install` wont work if the release already exists and just a regular `upgrade` command wont install a chart if a release does not exist.

**Review by mboersma — APPROVED** [review]

**Pull request merged**